test/livebook/live_markdown/markdown_helpers_test.exs

```diff
@@ -24,16 +34,56 @@ defmodule Livebook.LiveMarkdown.MarkdownHelpersTest do
       assert markdown == reformat(markdown)
     end
 
+    test "inline code with a line break" do
+      markdown = "The Game, `Mrs\nHudson`, is on!"
+      assert markdown == reformat(markdown)
+    end
+
+    test "inline code with extra spaces" do
+      markdown = "The Game, `Mrs Huds  on`, is on!"
+      assert markdown == reformat(markdown)
+    end
+
     test "combined" do
       markdown = "The Game, ~~***`Mrs Hudson`***~~, is on!"
       assert markdown == reformat(markdown)
     end
 
+    test "inline math" do
+      markdown = "The Game, $Mrs_{ij} Hudson_{ij}$, is on!"
+
+      assert markdown == reformat(markdown)
+    end
+
+    test "alternate brackets for inline math" do
+      markdown = "The Game, \\(Mrs_{ij} Hudson_{ij}\\), is on!"
+
+      assert markdown == reformat(markdown)
+    end
+
     test "link" do
       markdown = "The Game, [Mrs Hudson](https://youtu.be/M-KqaO1oH2E), is on!"
       assert markdown == reformat(markdown)
     end
 
+    test "link with IAL" do
+      markdown = "[link](url){: .classy}"
+
+      assert markdown == reformat(markdown)
```

Comment:
The annotation here is entirely discarded.

`earmark-parser` supports this style of adding classes to generated HTML; I do not know if Livebook _wants_ that functionality or not, but it seemed worth calling out explicitly (for instance, if files being imported use the feature)

Reply:
When converting AST to Markdown we generally ignore arbitrary attributes, so they don't end up in Livemarkdown and I don't think we should render IAL.